src/cordova.js

```diff
@@ -43,6 +43,80 @@ var m_window_removeEventListener = window.removeEventListener;
 var documentEventHandlers = {};
 var windowEventHandlers = {};
```

Comment:
These variables are being shadowed by the IIFE function, from lines 52 & 53.

I believe the intent for these to be removed since they are globally accessible via the `window` namespace (since they are replaced by the IIFE closure)